source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -11,7 +12,7 @@ import {
     StringWithSubstitutions,
     ContentTypes,
     IInput,
-    IResourceInformation
+    IResourceInformation,
```

Comment:
In the description, can you describe how these requirements are/or aren't addressed

```
<html>
<body>
<!--StartFragment-->

Event for when user clicks the next arrow on the carousel | Must
-- | --
Event for when a user clicks the previous arrow on the carousel | Must
Event for when a user navigates the carousel with the bottom dots | Must
Which carousel pages did the user see vs not see | Must
How much time a user spent on a carousel and which page | Must

<!--EndFragment-->
</body>
</html>
```